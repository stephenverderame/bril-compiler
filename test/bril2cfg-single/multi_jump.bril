@main {
    jmp .foo;
.foo:
    jmp .bar;
.bar:
    jmp .baz;
.baz:
.pizaz:
.wooo:
    x: int = const 42;
    jmp .caz;
.caz:
    print x;
}