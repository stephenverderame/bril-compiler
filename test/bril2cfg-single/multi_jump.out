digraph {
	subgraph cluster_main {
		label="main()";
		rankdir="TB";
		main_0 [label="START", shape="rectangle", style="rounded"];
		main_5 [label="x := 42\n", shape="rectangle", style="rounded"];
		main_7 [label="print x\n", shape="rectangle", style="rounded"];

		main_0 -> main_5;
		main_5 -> main_7;
	}
}
